### ISSUE TITLE
Airbnb Or Hotel?

### ISSUE THREAD

**Issue by honjy** [jsoma/playfair-projects#222]
I'd like to create a map, using what we've learned with Carto and QGIS, of Airbnb prices versus hotel prices in Singapore, just to see if it's better to rent an apartment on Airbnb or to book a hotel. The main challenge I foresee is that there isn't any data already available for Airbnb apartments in Singapore, so I may have to find a way to scrape that data myself.

If that is not possible I will stick to New York. I've already done a preliminary map below showing Airbnb prices across Manhattan, so you can see where the more expensive neighborhoods are.

My data is from [insideairbnb.com](insideairbnb.com). 

![listings_clean_1_by_honjy_08_14_2016_10_38_23](https://cloud.githubusercontent.com/assets/19495798/17654785/daf67180-6274-11e6-8c2d-5e88ed153aeb.png)
- [x] My pitch issue has `[Pitch]` in the title
- [x] My pitch issue has `[Data request]` in the title if I need additional data
- [x] My pitch issue links to my data set
- [x] My pitch issue explains what I'd like to explore in the data set
- [x] My pitch issue includes multiple possible headlines
- [x] My pitch issue includes any links/images I'm using as inspiration
- [ ] I have received two comments of peer feedback on my pitch issue

**Comment by playfairbot** [comment]
Hi there, I'm the Playfair Bot!

Would you mind **adding a category to your issue title?** Something like [Pitch] or [Story], or maybe [Meta] if it happens to be a bug. That way we can organize things nice and neat.

Thanks! :pray:

**Comment by barjacks** [comment]
That great, to compare them to hotels. I am pretty sure there won't be that much of a difference. What would also be interesting not just to compare price, but how many rooms certain users are offering. Are some of them actually bigger than hotels? I did something similar for Switzerland a while ago. And I ended up with a woman who, in terms of rooms on offer, was actually running the largest hotel in the [country](http://www.tagesanzeiger.ch/wirtschaft/unternehmen-und-konjunktur/Die-Managerin-des-groessten-Hotels-der-Schweiz/story/19763617).
